app/browser/api/ledger.js

```diff
@@ -2327,7 +2364,7 @@ const transitionWalletToBat = () => {
 
   try {
     client.transition(newPaymentId, (err, properties) => {
-      if (err) {
+      if (err || newClient == null) {
```

Comment:
I think you want `!newClient` here because otherwise `newClient`'s `false` could be used in the `else`.

Reply:
done